### ISSUE TITLE
MotionExporter not outputing trajectory speed

### ISSUE THREAD

**Issue by gustavoeb** [sebastianstarke/AI4Animation#11]
In the MotionExporter.ExportData(), the speed for each trajectory point is not being exported. But this data is used to create the input vector for the network in the BioAnimation_wolf.Animate(). This means when extracting new training data that the input vectors will not match in size.

From MotionExporter, line 372:
```
										for(int k=0; k<12; k++) {
											Vector3 position = current.Trajectory.Points[k].GetPosition().GetRelativePositionTo(current.Root);
											Vector3 direction = current.Trajectory.Points[k].GetDirection().GetRelativeDirectionTo(current.Root);
											Vector3 velocity = current.Trajectory.Points[k].GetVelocity().GetRelativeDirectionTo(current.Root);
											float[] style = current.Trajectory.Points[k].Styles;
											inputLine += FormatValue(position.x);
											inputLine += FormatValue(position.z);
											inputLine += FormatValue(direction.x);
											inputLine += FormatValue(direction.z);
											inputLine += FormatValue(velocity.x);
											inputLine += FormatValue(velocity.z);
											inputLine += FormatArray(style);
										}
```

From BioAnimation_wolf.cs, line 282:
```
			for(int i=0; i<PointSamples; i++) {
				Vector3 pos = GetSample(i).GetPosition().GetRelativePositionTo(currentRoot);
				Vector3 dir = GetSample(i).GetDirection().GetRelativeDirectionTo(currentRoot);
				Vector3 vel = GetSample(i).GetVelocity().GetRelativeDirectionTo(currentRoot);
				float speed = GetSample(i).GetSpeed();
				NN.SetInput(start + i*TrajectoryDimIn + 0, pos.x);
				NN.SetInput(start + i*TrajectoryDimIn + 1, pos.z);
				NN.SetInput(start + i*TrajectoryDimIn + 2, dir.x);
				NN.SetInput(start + i*TrajectoryDimIn + 3, dir.z);
				NN.SetInput(start + i*TrajectoryDimIn + 4, vel.x);
				NN.SetInput(start + i*TrajectoryDimIn + 5, vel.z);
				NN.SetInput(start + i*TrajectoryDimIn + 6, speed);
				for(int j=0; j<Controller.Styles.Length; j++) {
					NN.SetInput(start + i*TrajectoryDimIn + (TrajectoryDimIn - Controller.Styles.Length) + j, GetSample(i).Styles[j]);
```
				}
			}

**Comment by sebastianstarke** [comment]
True, but that is intended. I decided to not keep a hard-coded motion exporter for the quadruped demo, but instead keep updating it for my current research. I uploaded the processed final data for training (Input.txt and Output.txt) for the quadruped research, along with the source motion capture data. When processing the data for a specific project, those lines in the motion exporter need to be modified (i.e. specifying what shall be exported at which index). Note that this is irrelevant for the neural network python script, which simply gets arbitrary input and output data. It is just that I do not want to keep multiple copies of "dead" files in the project, which would keep accumulating quickly.
